third-parties/scroll/database/migrate/migrations/00003_l2_block.sql

```diff
@@ -0,0 +1,32 @@
+-- +goose Up
+-- +goose StatementBegin
+
+create table l2_block
+(
+    number                  BIGINT          NOT NULL,
+    hash                    VARCHAR         NOT NULL,
+    parent_hash             VARCHAR         NOT NULL,
+    header                  TEXT            NOT NULL,
```

Comment:
`l2_block` changed in https://github.com/scroll-tech/scroll/pull/631

Reply:
Will copy new migrations.

Reply:
Copy new migrations from scroll in commit [d76ab85](https://github.com/scroll-tech/rollup-explorer-backend/pull/141/commits/d76ab85511b81e8691fe8c706e55176da78a1b34). Will copy again after DB finalized.

Reply:
Probably finalized after https://github.com/scroll-tech/scroll/pull/633 merged.